### PR TITLE
WIP: get traceid from graph and display on deploy #trivial

### PR DESCRIPTION
**Pull request by JC1738**
screenshot from 2018-04-27 11-54-04
screenshot from 2018-04-27 11-53-46

### CODE REVIEW

**Comment by sous-machine-account** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">PR is classed as Work in Progress</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: cli/cli.go [hunk 1]
```diff
@@ -87,6 +89,7 @@ func (cli *CLI) Invoke(args []string) cmdr.Result {
 	}
 	reportInvocation(ls, start, args)
 	res := cli.CLI.Invoke(args)
+	res.SetTraceID(cli.CLI.TraceID)
```
Comment (JC1738) [inline cli/cli.go]: @nyarly I tried both cli.TraceID and cli.CLI.TraceID, none seem populated from Graph like it was for SousDeploy struct which did get it.  I am missing something, but not sure what it is, to force that population?

Comment (nyarly) [reply cli/cli.go]: You need to add the TraceID as a field on the constructor of the CLI in the `graph` package. c.f. https://github.com/samsalisbury/psyringe/blob/master/README.md#injection-types

Comment (JC1738) [reply cli/cli.go]: @samsalisbury @nyarly How do I find the constructor, was looking for it...  Sometimes they exist sometimes they seem to be done by Graph itself?

**Comment by JC1738** [comment]
Closing with a new direction of Scoop on Action() in main.go, will open a new one.

**Pull request closed without merging**